src/client/opamSolution.ml

```diff
@@ -422,7 +422,7 @@ let parallel_apply t
          (let spkgs = OpamSysPkg.Set.Op.(bypass -- !bypass_ref) in
           OpamConsole.note
             "Requirement for system package%s %s overridden in this switch. Use \
-             `opam option depext-bypass-=%s' to revert."
+             `opam option 'depext-bypass-=%S'' to revert."
```

Comment:
Are the single quotes really needed if we use `%S`?
```suggestion
             `opam option depext-bypass-=%S to revert."
```

Reply:
Yes, otherwise the shell removes the double quotes:
```shell-session
$ echo foo="bar"
foo=bar
```

Reply:
does this option really takes quotes? That's a bit surprising

Reply:
yes, it's based on the opam syntax I guess; I agree it's cumbersome — but all I did here was fix the message so that it printed an actually working command !